practice/package.json

```diff
@@ -13,8 +13,20 @@
   "license": "ISC",
   "devDependencies": {
     "@eslint/js": "^9.3.0",
+    "buffer": "^6.0.3",
+    "crypto-browserify": "^3.12.0",
     "eslint": "^9.3.0",
+    "events": "^3.3.0",
     "globals": "^15.3.0",
-    "parcel": "^2.12.0"
+    "os-browserify": "^0.3.0",
+    "parcel": "^2.12.0",
+    "path-browserify": "^1.0.1",
+    "process": "^0.11.10",
+    "stream-browserify": "^3.0.0",
+    "string_decoder": "^1.3.0",
```

Comment:
Why do you need to install more libs here? Please remove the libs that don't use